src/library_wasmfs_jsimpl.js

```diff
@@ -24,16 +24,18 @@ mergeInto(LibraryManager.library, {
     return wasmFS$backends[backend].freeFile(file);
   },
 
+  _wasmfs_jsimpl_write__deps: i53ConversionDeps,
   _wasmfs_jsimpl_write: function(backend, file, buffer, length, {{{ defineI64Param('offset') }}}) {
-    {{{ receiveI64ParamAsDouble('offset') }}}
+    {{{ receiveI64ParamAsI53('offset') }}}
```

Comment:
Is this change in name related to the added warning?

Reply:
Yes, `receiveI64ParamAsDouble` is a legacy function.   The new function that does basically this is called `receiveI64ParamAsI53`